pkg/features/kube_features.go

```diff
@@ -1044,6 +1051,8 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 
 	SecurityContextDeny: {Default: false, PreRelease: featuregate.Alpha},
 
+	SeparateTaintManager: {Default: false, PreRelease: featuregate.Alpha},
```

Comment:
I think we agreed to make it beta on day 1.

```suggestion
	SeparateTaintManager: {Default: true, PreRelease: featuregate.Beta},
```

Reply:
Thanks! I did missed this. The feature is now enabled by default as per your comment in a2f34ba4a9a